src/Symfony/Component/Messenger/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ CHANGELOG
 4.3.0
 -----
 
+ * Added optional `MessageCountAwareReceiverInterface` that receivers can implement
```

Comment:
maybe name it `MessageCountAwareInterface`? the `Receiver` part is not really needed, is it?